source/guide/index.md

```diff
@@ -6,99 +6,156 @@ order: 1.1
 
 ## Overview
 
-Here is an example writing in Weex and Vue.js:
+If you just want to try Weex, You don't need to install anything. There is an [online playground](http://dotwe.org/vue/) of Weex, you can write single page examples on it without any configurations. The source code should be writing in Vue.js [single file component](https://vuejs.org/v2/guide/single-file-components.html) syntax, and the render result of the web platform will be displayed in the mock phone shell.
+
+Here is an [example](http://dotwe.org/vue/8da01827631b21150a12dd54d7114380) written in Weex and Vue.js:
 
 ![Weex Example](./images/weex-example-yo.png)
 
-This example is written in Vue.js [single file component](https://vuejs.org/v2/guide/single-file-components.html) syntax. Vue.js 2.0
+This example rendered a latter "Yo" in the center of the screen. If you want to preview the render result on the real mobile device, you could install the [Weex playground app](https://weex-project.io/playground.html) or integrate Weex SDK into your own app. And then, scan the QR code on the right of the online playground you will see the result rendered on your own device.
+
+Within the `<template>` of the source code, You must be familiar with the `<div>` which is widely used on the web, and it's also the generic container on Weex. But the `<text>` component is provided by Weex, it's a block-level text container.
+
+> The raw text node can only be placed in the `<text>` component, otherwise, it will be ignored.
+
+Within the `<style>` tag, you can write CSS to describe the styles of the component, and the styles are [**scoped**](https://vue-loader.vuejs.org/en/features/scoped-css.html) by force on Weex.
+
+~~[Here is more examples written in Weex and Vue.js](https://hanks10100.github.io/weex-vue-examples/)~~.
+
+### Native Components
+
+In the example above, the `<div>` and the `<text>` is rendered into corresponding native views on the mobile device, not `HTMLElement`.
+
+![Native Components](./images/native-component.png)
+
+Weex implemented render engines in each end and provides a group of [built-in components](../references/components/) for basic usage. Based on these components, you can compose and wrap more custom components.
+
+Although the components in Weex look like HTML tags, you can't use all HTML tags, you can only use the built-in components and your custom components.
+
+Under the scene, Weex is using the native widgets. Although Weex emphasize the consistency of each end, we still embrace the platform's own behavior and UI differences. For example, [this `<switch>` component](http://dotwe.org/vue/d96943452b6708422197c47920903823) may look different on Android and iOS.
+
+![Different switch](./images/different-switch.png)
+
+If you want to use more native component other than the built-in components, you should have to implement it on each platform and be consistent. The most practical way is to integrate the existing native components to weex platform.
```

Comment:
you need to implement them on each platform and keep their behaviors to be consistent.